projects/packages/scheduled-updates/src/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-update-schedules.php

```diff
@@ -78,6 +80,19 @@ public function register_routes() {
 			)
 		);
 
+		register_rest_route(
+			$this->namespace,
+			'/' . $this->rest_base . '/last-status',
```

Comment:
What do you think of `status` for this one? Or `result`?

I understand why it's not a single entity endpoint (`Jetpack_Autoupdate` doesn't know which schedule is running), but it really feels like it should be :) I'll think about it over the weekend

Reply:
Probably `result` is better. Because it is not just the "status".